_posts/2015-06-23-wikidata-akka-streams.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: post
+title: Process the whole Wikidata in 7 minutes with your laptop (and Akka streams)
+date: 2015-06-23 11:20:32
+excerpt: How Akka Streams can be used to process the Wikidata dump in parallel and using constant memory with just your laptop.
+image: /assets/images/wikidata-akka-streams.jpg
+thumbnail: /assets/images/wikidata-akka-streams-thumbnail.jpg
+categories: engineering
+author: Albert Pastrana
+---
+
+Here at Intent HQ we use Wikipedia and Wikidata as one of our sources of data. It is very important because they both encode an enormous amount of information in several languages that we use to build our [Topic Graph](https://www.intenthq.com/the-topic-graph/).
```

Comment:
I agree with @adam1128nz. Also one final one from me on this line, I suspect "we use Wikipedia and Wikidata as one of our sources of data" should be "we use Wikipedia and Wikidata as sources of data".